index.js

```diff
@@ -117,6 +109,12 @@ export default class TimeInput extends React.Component {
 				)
 			}
 		}, 0)
+
+		if (this.props.forcePolyfill || !supportsTime) {
+			loadPolyfill(() => {
+				this.onPolyfillLoad(window.timePolyfillHelpers)
```

Comment:
`onPolyfillLoad` wasn't attached to the `this` object. It was attached to the `<input>` element. That might also be the cause.

Reply:
...or maybe not. I'm a little confused :/

Reply:
you pushed `this` to array, and then interated through it calling `onPolyfillLoad`

Reply:
i kinda have slow internet, so It'll take some time to download VM

Reply:
I think I'll just wait for you to take a look at it after the VM has finished downloading.